kmir/src/kmir/kmir.py

```diff
@@ -110,8 +110,11 @@ def rpc_session(self, server: KoreServer, claim_id: str, trace_rewrites: bool =
                 )
 
     # A wrapper on KProve's get_claims
-    def get_all_claims(self, spec_file: Path) -> list[KClaim]:
-        return self.mir_prove.get_claims(spec_file)
+    def get_all_claims(self, spec_file: Path, claim_label: Optional[str]) -> list[KClaim]:
```

Comment:
```suggestion
    def get_all_claims(self, spec_file: Path, claim_label: str | None) -> list[KClaim]:
```

Reply:
Also, `get_claims` already supports a potentially empty list, so we probably don't need this function.

Reply:
`get_all_claims()` is a simple wrapper on pyk's implementation to make KMIR's code succinct.
Direct use of pyk's function would make the calling path too long. 

I'd prefer the current implementation.